hack/build-opm-catalog.sh

```diff
@@ -0,0 +1,384 @@
+#!/bin/bash -x
+
```

Comment:
I'd add the source of this file in case it drifts

```suggestion
# Vendored from https://github.com/openshift/boilerplate/blob/ed5706ecedc236099532a0edb2c5a37e41e0235f/boilerplate/openshift/golang-osd-operator/build-opm-catalog.sh
```